kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue

```diff
@@ -379,8 +374,9 @@
       handleSeek() {
         // record progress before updating the times,
         // to capture any progress that happened pre-seeking
-        this.recordProgress();
-
+        if (!this.forceTimeBasedProgress) {
```

Comment:
I don't think we need these `forceTimeBasedProgress` checks in this file - as we are only doing time based progress here.